python/Ganga/Core/GangaRepository/SessionLock.py

```diff
@@ -533,7 +533,7 @@ def session_read(self, fn):
             # This can fail (thats OK, file deleted in the meantime)
             fd = None
             try:
-                fd = os.open(fn, os.O_RDONLY)
+                fd = os.open(fn, os.O_RDWR)
```

Comment:
@egede I'm just curious why we have a problem opening a file for reading when we touch it compared to opening a file for write only when we read the lock state.
If there are known problems on filesystems we support I think it's probably cleaner to just open _everything_ r/w throughout the file as it reduces the questions going forward of 'why was this one not r/w'. I'm not sure there are is a performance difference in opening a file for r/w against ro but I've never looked tbh.